src/styles/index.scss

```diff
@@ -2,2 +1,134 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      max-width: 160px;
+      max-height: 134px;
+      object-fit: contain;
+    }
+  }
+
+  &__model {
+    margin-top: 40px;
+    width: 166px;
+    height: 36px;
+    font-size: 12px;
+    font-weight: 500;
+    padding-top: 0;
+    color: #060b35;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    align-self: start;
+    width: 98px;
+    height: 14px;
+
+    &-style {
+      white-space: nowrap;
+      padding-left: 17px;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+      margin: 0;
+    }
+  }
+
+  &__grade {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-stars {
+      box-sizing: border-box;
+      width: 96px;
+      height: 16px;
+
+      &-list {
+        list-style: none;
+        justify-content: space-between;
+        margin: 1px;
+
+        &:nth-child(n + 1) {
```

Comment:
The `&:nth-child(n + 1)` selector seems redundant here since it applies to all child elements, effectively having no specific selection criteria. You can remove this selector and apply the styles directly to `&-list` if there are no other children that the styles should not apply to.